bgpd/bgp_evpn_vty.c

```diff
@@ -675,7 +676,13 @@ static void show_vni_routes(struct bgp *bgp, struct bgpevpn *vpn, int type,
 			if (json)
 				json_path = json_object_new_array();
 
-			route_vty_out(vty, &rn->p, pi, 0, SAFI_EVPN, json_path);
+			if (detail)
+				route_vty_out_detail(vty, bgp, &rn->p, pi,
+						     AFI_L2VPN, SAFI_EVPN,
+						     json_path);
+			else
+				route_vty_out(vty, &rn->p, pi, 0, SAFI_EVPN,
```

Comment:
why 0 is passed instead of AFI_L2VPN here ? AFI_L2VPN is passed in case of details output.

Reply:
Like in PR 4116, it is not AFI. Let me check on why AFI is not required here.

Reply:
The API seems to assume AFI being L2VPN and we are all good here.